src/radical/pilot/agent/scheduler/base.py

```diff
@@ -840,6 +840,10 @@ def _try_allocation(self, unit):
         self._change_slot_states(slots, rpc.BUSY)
         unit['slots'] = slots
 
+        env_dict = unit['description'].get('environment', {})
+        env_dict['NODE_LFS_PATH'] = slots['lfs_per_node']['path']
```

Comment:
We should prefix all variables we set with `RP_` - at least that is the scheme we do elsewhere, in order to minimize chances to collide with user or system environment.  Maybe `RP_LSF_PATH`?  (`node` and `local` (from LSF) are redundant anyway).

Reply:
I would agree, let's open a ticket for it and introduce backward compatibility for a bit. Then we can change it to.

Reply:
Makes sense - will you take care of the issue creation, please?